README.md

```diff
@@ -121,9 +121,11 @@ OpenConsole.sln may be built from within Visual Studio or from the command-line
 
 ```shell
 nuget restore OpenConsole.sln
-msbuild OpenConsole.sln
+msbuild OpenConsole.sln /p:Platform=x64
 ```
 
+(Replace "x64" with another platform if desired.)

```

Comment:
Can you list the other two platform options we support instead of the generic catch all? 

It's 'arm64' and then whatever the actual platform configuration tag is for 32-bit (it's either win32, x86 or something, check and put that there, please.)

For example
`(Replace "x64" with either "arm64" or "x86" if desired.)`